in_toto/in_toto_record.py

```diff
@@ -172,6 +179,14 @@ def main():
       "Path to GPG keyring to load GPG key identified by '--gpg' option.  If"
       " '--gpg-home' is not passed, the default GPG keyring is used."))
 
+  parent_parser.add_argument("--exclude", dest="exclude_patterns",
+      required=False, metavar="<pattern>", nargs="+", help=(
+      "Do not record 'materials/products' matched by the passed exclude"
+      " patterns. Passed exclude patterns override previously set patterns,"
+      " using e.g.: environment variables or RCfiles. See"
+      " ARTIFACT_EXCLUDE_PATTERNS documentation for additional info."
```

Comment:
I just realized that we need to repeat ourselves on these. I wonder if moving both strings to the same place would be worth it.

Reply:
100% agreed. We should create a ticket for generalizing our cli tools. There are a lot of shared arguments especially in `in-toto-run` and `in-toto-record`. And all tools take singing or verification key arguments for securesystemslib keys and gpg keys.